lib/std/math/big/int.zig

```diff
@@ -19,6 +19,28 @@ const native_endian = builtin.cpu.arch.endian();
 
 const debug_safety = false;
 
+// SAFETY: the caller must ensure it won't overflow a Limb
+// parse at most `max_n` digits
+fn sliceToLimb(base: u8, string: []const u8, max_n: usize, _i: *usize) !?Limb {
```

Comment:
Mixing `_i` and `i` make things less readable. Please use for the function arguments a bit more descriptive names or describe in a comment what "better replacement of `_i`" is.

Reply:
maybe `slice_index` ?

Reply:
As I understand it, `_i` is the start of the slice, so `string_i` + a comment that `string_i` is the slice_start index into the string sounds better. Not sure, if this could be made more dense.